src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -317,7 +321,9 @@ private static <E extends Enum<E>> List<Object> getEnumPossibleValues(Class<E> e
         new Parameter(NEWTON_KRYLOV_LINE_SEARCH_PARAM_NAME, ParameterType.BOOLEAN, "Newton Krylov line search activation", NewtonKrylovParameters.LINE_SEARCH_DEFAULT_VALUE),
         new Parameter(REFERENCE_BUS_SELECTION_MODE_PARAM_NAME, ParameterType.STRING, "Reference bus selection mode", ReferenceBusSelector.DEFAULT_MODE.name(), getEnumPossibleValues(ReferenceBusSelectionMode.class)),
         new Parameter(WRITE_REFERENCE_TERMINALS_PARAM_NAME, ParameterType.BOOLEAN, "Write Reference Terminals", WRITE_REFERENCE_TERMINALS_DEFAULT_VALUE),
-        new Parameter(VOLTAGE_TARGET_PRIORITIES_PARAM_NAME, ParameterType.STRING_LIST, "Voltage target priorities for voltage controls", LfNetworkParameters.VOLTAGE_CONTROL_PRIORITIES_DEFAULT_VALUE, getEnumPossibleValues(VoltageControl.Type.class))
+        new Parameter(VOLTAGE_TARGET_PRIORITIES_PARAM_NAME, ParameterType.STRING_LIST, "Voltage target priorities for voltage controls", LfNetworkParameters.VOLTAGE_CONTROL_PRIORITIES_DEFAULT_VALUE, getEnumPossibleValues(VoltageControl.Type.class)),
+        new Parameter(TRANSFORMER_VOLTAGE_CONTROL_STABLE_PARAM_NAME, ParameterType.BOOLEAN, "Maintain initial tap position if possible", LfNetworkParameters.TRANSFORMER_VOLTAGE_CONTROL_STABLE_DEFAULT_VALUE),
+        new Parameter(TRANSFORMER_VOLTAGE_CONTROL_THT_LIMIT_PARAM_NAME, ParameterType.INTEGER, "Limit in KV below wich group stop controlling tension during tap computation. -1 - the default - for automatic determination.", LfNetworkParameters.TRANSFORMER_VOLTAGE_CONTROL_THT_LIMIT_DEFAULT_VALUE)
```

Comment:
use `voltage` instead of `tension`

Reply:
replaced by noex equivalent parameter in mail.